apps/site/src/app/page.tsx

```diff
@@ -1,7 +1 @@
-export default function Home() {
-	return (
-		<main>
-			<h1>ZotHacks 2023</h1>
-		</main>
-	);
-}
+export { Landing as default } from "@/views"; 
```

Comment:
Question: any particular reason you renamed the component from "Home" to "Landing"?